virttest/utils_windows/drive.py

```diff
@@ -37,3 +39,93 @@ def get_floppy_drives_letter(session):
     """
     cond = "MediaType!=0 AND MediaType!=11 AND MediaType!=12"
     return _logical_disks(session, cond=cond, props=["DeviceID"])
+
+
+def rescan_disk(session):
+    """
+    Rescan disk in windows guest.
+
+    :param session: Session object.
+    """
+    rescan_cmd = "echo rescan > cmd"
+    rescan_cmd += " && echo exit >> cmd"
+    rescan_cmd += " && diskpart /s cmd"
+    session.cmd(rescan_cmd)
+
+
+def get_drive_volume_id(session, did, disk_letter):
```

Comment:
After read this function, I'd suggest the interface to be:
```python
def get_drive_volume_num(session, drive_letter, disk_index=None):
    ...
    return volume_num
```

Reply:
thanks, updated~